recipes/libusb/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "libusb" %}
+{% set version = "1.0.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/libusb/libusb/releases/download/v{{ version}}/{{ name }}-{{ version }}.tar.bz2'
+  sha256: 75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libusb') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('libusb') }}  # [linux]
+    - libtool  # [unix]
```

Would add an empty `host` to separate the `build` and `host` environments to avoid clobbering.

```suggestion
    - libtool  # [unix]
  host:
```

---

The linter would then produce an error message, see https://github.com/conda-forge/staged-recipes/pull/8737#issuecomment-515139692